pkg/service/resource/zone.go

```diff
@@ -156,7 +156,7 @@ func (r *resource) GetEnvCnt() (cnt int) {
 func (r *resource) GetSelectData() (respRegion, respZone, respEnv []view.SelectData) {
 	// 找到所有的Region
 	var regionArr []db.Zone
-	r.DB.Model(db.Zone{}).Group("region_code").Find(&regionArr)
+	r.DB.Model(db.Zone{}).Select("region_name, region_code").Group("region_name, region_code").Find(&regionArr)
```

Comment:
The code line should  not be so long. :XD

Reply:
already fixed in next commit